darts/models/forecasting/pl_forecasting_module.py

```diff
@@ -29,10 +30,12 @@ def __init__(
         input_chunk_length: int,
         output_chunk_length: int,
         loss_fn: nn.modules.loss._Loss = nn.MSELoss(),
+        torch_metrics: Optional[List[str]] = None,
+        metrics_params: Optional[List[Dict]] = None,
```

Comment:
I believe that the class based metrics (see my comment from above) don't require any additional parameters at metric computation. They are already given at object creation.

If so, we could also remove this parameter